ChatQnA/benchmark/performance/kubernetes/intel/gaudi/generate_helm_values.py

```diff
@@ -38,6 +38,7 @@ def generate_helm_values(with_rerank, num_nodes, hf_token, model_dir, node_selec
         values["teirerank"] = {"nodeSelector": {key: value for key, value in node_selector.items()}}
     else:
         values["image"] = {"repository": "opea/chatqna-without-rerank"}
+        values["teirerank"] = {"enabled": "false"}
```

Comment:
Please change from **false** to **False**. Otherwise, yaml file will have the followings generated which is not recognized by helm chart deploy.
```
teirerank:
  enabled: 'false'
```

Reply:
I see, thanks, will update it.